util.go

```diff
@@ -64,6 +68,42 @@ func init() {
 		AwaitingPulseJobUpdateStates[status] = true
 	}
 }
+func createCertPool(path string, extensions map[string]struct{}) (*x509.CertPool, error) {
+	certPool := x509.NewCertPool()
+
+	_, err := os.Stat(path)
+	if err != nil {
+		return nil, errors.New("given certs path doesn't exist")
```

Comment:
why not include the `err` returned
```
errors.Wrap(err,"given certs path doesn't exist")
```

Reply:
The error message will be more informative if it contains the path. Mia's suggestion may resolve that.

Reply:
Good catch, wrapped the error now.